### PR TITLE
Add samples for PITR

### PR DESCRIPTION
**Pull request by zoercai**

### CODE REVIEW

File: samples/samples/backup_sample.py [hunk 1]
```diff
@@ -47,8 +48,8 @@ def create_backup(instance_id, database_id, backup_id):
     # Get the name, create time and backup size.
     backup.reload()
     print(
-        "Backup {} of size {} bytes was created at {}".format(
-            backup.name, backup.size_bytes, backup.create_time
+        "Backup {} from {} of size {} bytes was created at {}".format(
```
Comment (thiagotnunes) [inline samples/samples/backup_sample.py]: Could we change the message to be:

```
Backup {} of size {} bytes was created at {} for version of database at {}
```

File: samples/samples/backup_sample.py [hunk 2]
```diff
@@ -75,10 +76,11 @@ def restore_database(instance_id, new_database_id, backup_id):
     new_database.reload()
     restore_info = new_database.restore_info
     print(
-        "Database {} restored to {} from backup {}.".format(
+        "Database {} restored to {} from backup {} with version time of {}.".format(
```
Comment (thiagotnunes) [inline samples/samples/backup_sample.py]: Could we remove the "of" from the sentence:

```
Database {} restored to {} from backup {} with version time {}
```

File: tests/system/test_system.py [hunk 3]
```diff
@@ -711,7 +711,9 @@ def test_backup_workflow(self):
         self.to_drop.append(database)
         operation = database.restore(source=backup)
         restored_db = operation.result()
-        self.assertEqual(self.database_version_time, restored_db.restore_info.backup_info.create_time)
```
Comment (thiagotnunes) [inline tests/system/test_system.py]: Is this change necessary in this PR?

**Review by thiagotnunes — COMMENTED** [review]
Added a few message changes, so that the samples are consistent across languages.

Could you also add the new sample: `createDatabaseWithVersionRetentionPeriod` into the PR?

File: tests/system/test_system.py (hunk 3 above)
Comment (thiagotnunes) [reply tests/system/test_system.py]: I don't mind if it is, just wondering if you were fixing a style error.

Comment (zoercai) [reply tests/system/test_system.py]: It's not necessary, it was just part of the automatic linting, I've removed it locally

**Review by thiagotnunes — APPROVED** [review]
LGTM

**Review by thiagotnunes — APPROVED** [review]
